server/sonar-server/src/test/java/org/sonar/server/component/ws/SearchProjectsActionTest.java

```diff
@@ -386,19 +447,13 @@ private SearchProjectsWsResponse call(SearchProjectsRequest.Builder requestBuild
     SearchProjectsRequest wsRequest = requestBuilder.build();
     TestRequest httpRequest = ws.newRequest()
       .setMediaType(MediaTypes.PROTOBUF);
-
-    String organization = wsRequest.getOrganization();
-    if (organization != null) {
-      httpRequest.setParam(PARAM_ORGANIZATION, organization);
-    }
-    httpRequest.setParam(Param.PAGE, String.valueOf(wsRequest.getPage()));
-    httpRequest.setParam(Param.PAGE_SIZE, String.valueOf(wsRequest.getPageSize()));
-    String filter = wsRequest.getFilter();
-    if (filter != null) {
-      httpRequest.setParam(PARAM_FILTER, filter);
-    }
-    httpRequest.setParam(Param.FACETS, Joiner.on(",").join(wsRequest.getFacets()));
-
+    ofNullable(wsRequest.getOrganization()).ifPresent(organization -> httpRequest.setParam(PARAM_ORGANIZATION, organization));
```

Comment:
I think the `setNullable` method should be extract to not be specific to Protobuf